UKHomeOffice/sgmr-service
# 1.1.0 - SPCR-148 Update interaction & data capture for departure and arrival ports

## Description
We need to ensure we pass the port name and unlocode to the API in the correct format so that known ports are recognised by CBP, to provide the best chance of having an accurate departure and arrival location

**PortField tests are incomplete and have been raised as a separate ticket.**

## To Test
- Reinstall packages
- Go the departure form
- Type in the autocomplete field
- It should bring you a list of matches you can select
- Click "I cannot find the location in this list"
- You should see an input field appear
- Start typing in the input field
- The autocomplete selection should be cleared
- Reselect an option after typing in the autocomplete field
- The other field's input should be cleared
- Fill the rest of the page and continue to the arrival form
- Repeat the steps carried out in the departure form, expected results should be the same
- Page layout and content should match the design from the ticket

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
